README.md

```diff
@@ -132,10 +264,69 @@ which **features** get built **next**_!
 
 ## How?
 
+If you want to get involved in helping us with the App,
+there are 4 ways to help:
+
+1. **Use** the **_Beta_ App**!
+`[insert app link here!]`
+2. Give us ***feedback*** on the **_existing_ UX**
+`[insert feedback link here!]`
+3. **Read** the
+[issues](https://github.com/dwyl/time/issues?q=label%3A%22help+wanted%22+is%3Aissue+is%3Aopen)
+and help us answer questions if you can.
+4. Write some **`code`**!
+See: https://github.com/dwyl/technology-stack
+
+
+### Tech & Features Roadmap
+
+
 This list _will_ evolve over time. <br />
 We will insert links to specific features
 and check them off this list as we go.
 
+#### MVP Basic Workflow
+
++ [ ] Mobile First UI/UX to **Capture** **`plaintext`**
++ [ ] Save **`plaintext`** data to Phoenix backend
++ [ ] Simple UX to **Categorise** (Transform)
+the **`plaintext`** into actionable items. <br />
+UI/UX **`help wanted`**: https://github.com/dwyl/product-roadmap/issues/15
++ [ ] Start working on task (_start timer for item_)
+see: https://github.com/nelsonic/time-mvp-phoenix#create-schemas
++ [ ] Check a task off as done.
+
+After the MVP workflow is complete,
+we will hold a sprint demo to showcase the state of the App.
+The sprint demo will be recorded on our Zoom call.
+Following the sprint demo call
+we will do a backlog grooming session to discuss the features/ideas list:
+
+#### Offline Support
+
++ [ ] Create Progressive Web App using Elm.
+see:
```

Comment:
```suggestion
```

Or did you have a link in mind for PWA with Elm?

Reply:
@SimonLab good catch. The link is: https://github.com/dwyl/learn-elm/issues/54

Reply:
```suggestion
see: dwyl/learn-elm#54
```

Reply:
@simonlab thanks for adding the reference. it would be more effective if it were a hyperlink.
`[dwyl/learn-elm#54](https://github.com/dwyl/learn-elm/issues/54)`

Reply:
Yes indeed. I thought Github was doing it automatically :thinking: 
```suggestion
see: [dwyl/learn-elm#54](https://github.com/dwyl/learn-elm/issues/54)
```